core/examples/luigi-sample-angular/e2e/tests/navigation.spec.js

```diff
@@ -199,5 +199,43 @@ describe('Navigation', () => {
       cy.get('.no-side-nav').should('exist');
       cy.get('.fd-app__sidebar').should('not.be.visible');
     });
+
+    it('Open navigation node in a modal', () => {
+      let iframeBody;
+      cy.get('iframe', { timeout: 1000 }).then(ifr => {
```

Comment:
Looks like these timeout is not needed (run the test a couple of times locally)

Reply:
ok